### ISSUE TITLE
Inability to disable Previous / Next tab behavior on mouse buttons 3 and 4 (side buttons)

### ISSUE THREAD

**Issue by cbj4074** [notepad-plus-plus/notepad-plus-plus#3496]
### Description of the Issue
By default, mouse buttons 3 and 4, which are most common on ball-mice (e.g., Logitech brand), are bound to the Previous and Next tab functions in Notepad++.

In other words, clicking button 3 (left) activates the previous document tab, and clicking button 4 (right) activates the next tab.

This behavior is problematic for ball-mice because button 3 is often used for Universal Scroll (which is also broken in Notepad++, but that's another matter). That is, holding-down button 3 engages Universal Scroll, so to scroll through a document, one typically holds-down button 3 and spins the mouse-ball.

This doesn't work in Notepad++, because attempting to hold-down button 3 causes Notepad++ to switch to the previous tab.

### Steps to Reproduce the Issue
1. Use ball-mouse that has buttons 3 and 4, e.g., http://support.logitech.com/en_us/product/trackman-marble
2. Install Logitech SetPoint software and set buttons 3 and 4 to "Generic Button" on the "Button Settings" tab. (*Note*: I set these buttons to "Generic Button" because Logitech's Universal Scroll behavior is rubbish, and I'm forced to use an alternative, https://github.com/Seelge/TrackballScroll , which requires "Generic Button" to work correctly.)
3. Open Notepad++, open multiple tabs, and click button 3 or 4. Notice that tab-switching occurs.

SetPoint is configured as follows:

![image](https://user-images.githubusercontent.com/1236883/27960444-51516300-62f9-11e7-811c-aaba0f71cc58.png)

### Expected Behavior
I expect to be able to disable these mouse button bindings so that I can use button 3 and my mouse-ball to scroll in Notepad++.

### Actual Behavior
I cannot find any means by which to disable the mouse button 3 and 4 bindings.

### Debug Information
Notepad++ v7.4.2   (32-bit)
Build time : Jun 18 2017 - 23:34:19
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : ON
Local Conf mode : OFF
OS : Windows 7 (64-bit)
Plugins : mimeTools.dll NppConverter.dll NppExport.dll NppFTP.dll NppTextFX.dll PluginManager.dll 

### Other Info
I tried removing the keyboard bindings for the Previous and Next tab shortcuts, but doing so has no effect on the mouse button bindings.

**Comment by cbj4074** [comment]
I have no idea what changed, but the desired scrolling behavior now works as expected for me in Notepad++!

All I can think of is that I rebooted the machine for the first time in a while. I can't think of why that would have any relevance, though, because changes made using the Logitech SetPoint utility seem to take effect in real-time.

I will keep an eye on it and re-open the issue if it recurs!

**Comment by cbj4074** [comment]
I finally figured out what was causing this. Notepad++ has a "Launch as Administrator" mode (which I love, by the way!), and it was only when launched as with Administrator privileges that I experienced this problem.

This makes perfect sense, as the mouse settings are likely user-account-specific, and the TrackballScroll executable runs under my own account (not Administrator).

At this point, I'm not sure whether the problem occurs due to mouse buttons 3 and 4 not being assigned "Generic Button", or TrackballScroll not running, or both. I'm also not sure what effect (if any) unsetting the Next and Previous page hotkeys in Notepad++ may have had.

Just glad to have it solved! 👍
